package.json

```diff
@@ -26,7 +26,7 @@
     "@sheerun/mutationobserver-shim": "^0.3.3",
     "@testing-library/jest-dom": "^5.3.0",
     "@testing-library/react": "^10.0.2",
-    "@theforeman/builder": "^4.6.0",
+    "@theforeman/builder": "^6.0.0",
```

Comment:
Should this be `>= 6.0.0` to ensure it doesn't break on the next update?

Reply:
we could change to that for builder for now. Ideally we don't have to update all the foreman-js metapackages at the same time so a major version change in `@theforeman/builder` did mean a breaking change. I'm opening up a discussion on this topic.

Reply:
@tbrisker updated to use `>=`